app/controllers/KairosDbProxyController.java

```diff
@@ -192,7 +192,12 @@ public KairosDbProxyController(
                 final Optional<Aggregator> aggregatorWithSampling = metric.getAggregators().stream().filter(
                         aggregator -> aggregator.getSampling().isPresent()).findFirst();
                 newAggregators.add(aggregatorWithSampling.map(
-                        aggregator -> ThreadLocalBuilder.clone(aggregator, Aggregator.Builder.class, b -> b.setName("merge"))).
+                        aggregator -> ThreadLocalBuilder.build(Aggregator.Builder.class, b -> b
```

Comment:
This might be easier to read as a full lambda block:

```
ThreadLocalBuilder.build(Aggregator.Builder.class, b -> {
    b.setName("merge");
    aggregator.getSampling.ifPresent(p -> b::setSampling)
     ...
  })
```